docs/resources/supportedChains.md

```diff
@@ -38,14 +38,16 @@ Don't see a blockchain you want?  Fill out this form for EVM chains and we'll ad
 - hyperspace
 - scrollAlphaTestnet
 - zksync
+- litprotocol
```

Comment:
should we be referring to Chronicle here?

Reply:
somehow i see this change in Chris's PR too. Perhaps this PR branch is outdated?

Reply:
this part is autogenerated from a script, tho likely this PR may be out of date if it's in Chris' PR and here